torch/nn/modules/linear.py

```diff
@@ -101,8 +101,6 @@ def extra_repr(self) -> str:
 # This class exists solely for Transformer; it has an annotation stating
 # that bias is never None, which appeases TorchScript
 class _LinearWithBias(Linear):
-    bias: Tensor
```

Comment:
This seems incorrect, see the comment above it. I'm guessing this isn't tested with TorchScript, and hence the test suite is green. Or it is tested, in which case the comment is outdated. Either way, this needs investigating and mypy needs to be made happy a different way if the comment here is correct.

Reply:
The comment is at least partially wrong, because `_LinearWithBias` is used in `MultiheadAttention`, which is also public API and used elsewhere than just `Transformer`.

Reply:
`assert bias is not None` or `# type: ignore` seems like a safer fix here, to not accidentally break some `torchscript`'ed code.